src/pages/Checklist/index.tsx

```diff
@@ -483,6 +513,11 @@ export const Checklist = () => {
                 />
               </Text>
             </li>
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="At time of the Merge, third-party providers (such as Infura and Alchemy) will no longer be viable options to outsource execution layer responsibilities. All stakers must run a pair of both an execution and consensus client to properly attest to the network." />
```

Comment:
Should we use the present tense for the post-merge scenario?
```suggestion
                <FormattedMessage defaultMessage="After the Merge, third-party providers (such as Infura and Alchemy) are no longer viable options to outsource execution layer responsibilities. All stakers must run a pair of both an execution and consensus client to properly attest to the network." />
```

Reply:
alternatively, just to make it very clear we are alreayd post-merge...
```suggestion
                <FormattedMessage defaultMessage="Since the Merge, third-party providers (such as Infura and Alchemy) are no longer being viable options to outsource execution layer responsibilities. All stakers must run a pair of both an execution and consensus client to properly attest to the network." />
```

Reply:
Good catch